paper/paper.md

```diff
@@ -1,124 +1,239 @@
-contains the ``Location`` and ``PVSystem`` classes. These abstractions
-provide simple methods that wrap the core function API layer. The method
-API simplification is achieved by separating the data that represents
-the object (object attributes) from the data that the object methods
-operate on (method arguments). For example, a ``Location`` is
-represented by a latitude, longitude, elevation, timezone, and name,
-which are ``Location`` object attributes. Then a ``Location`` object
-method operates on a ``datetime`` to get the corresponding solar
-position. The methods combine these data sources when calling the
-function layer, then return the results to the user. The final level of
-API is the ``ModelChain`` class, designed to simplify and standardize
-the process of stitching together the many modeling steps necessary to
-convert a time series of weather data to AC solar power generation,
-given a PV system and a location.
-
-pvlib python was ported from the PVLib MATLAB toolbox in 2014
-[@Stein2012, @Andrews2014]. Efforts to make the project more pythonic
-were undertaken in 2015 [@Holmgren2015]. Additional features continue to
-be added, see, for example [@Stein2016, @Holmgren2016] and the
-documentation's "What's New" section.
-
-pvlib python has been used in numerous studies, for example, of solar
-power forecasting [@Gagne2017, @Holmgren2017], development of solar
-irradiance models [@Polo2016], and estimation of photovoltaic energy
-potential [@Louwen2017]. Mikofski et. al. used pvlib python to study
-the accuracy of clear sky models with different aerosol optical depth
-and precipitable water data sources [@Mikofski2017] and to determine the
-effects of spectral mismatch on different PV devices [@Mikofski2016].
-pvlib python is a foundational piece of an award, "An Open Source
-Evaluation Framework for Solar Forecasting," made under the Department
-of Energy Solar Forecasting 2 program [@DOESF2].
-
-Plans for pvlib python development includes the implementation of new
-and existing models, addition of functionality to assist with
-input/output, and improvements to API consistency.
-
-The source code for each pvlib python version is archived with Zenodo
-[@pvlibZenodo].
 
-# Acknowledgements
-
-The authors acknowledge and thank the code, documentation, and
-discussion contributors to the project.
+# Summary
 
-WH acknowledges support from the Department of Energy's Energy
-Efficiency and Renewable Energy Postdoctoral Fellowship Program
-(2014-2016), Tucson Electric Power, Arizona Public Service, and Public
-Service Company of New Mexico (2016-2018), and University of Arizona
-Institute for Energy Solutions (2017-2018).
+pvlib python is a community-developed, open-source software toolbox
+for simulating the performance of solar photovoltaic (PV) energy
+systems.  It provides peer-reviewed reference implementations of
+over 100 empirical and physics-based models from the scientific literature,
```

Comment:
```suggestion
systems.  It provides reference implementations of
over 100 empirical and physics-based models from the peer-reviewed scientific literature,
```

Reply:
I think it's a shame that it no longer states that contributions to the package are peer-reviewed (as in our GitHub review and approval procedure). Perhaps we can add a new sentence stating that we employ rigorous peer-review of all contributions to the package?

Reply:
I suggested moving the term because (to me) peer review means something specific - blind review refereed by a third party. We aren't doing that in pvlib, although all contributions go through some kind of review. Maybe we add "openly-reviewed" to the list of best practices in the next paragraph?

Reply:
The peer review of this JOSS article will be open FYI ;)

But I see your point. I think our review often excels over regular peer reviews of journal articles. As long as it is mentioned in some form I am happy.